modules/oncodrivefml/1.0/config/default.yaml

```diff
@@ -0,0 +1,70 @@
+lcr-modules:
+    
+    oncodrivefml:
+        
+        inputs:
+            # Available wildcards: {seq_type} {genome_build}
+            input_maf: "__UPDATE__"
+            subsetting_categories: "__UPDATE__"
+
+        genome_builds: ["__UPDATE__"]
+        # Available regions options: cds, lincrna, 3utr, 5utr, promoter_splice_sites
```

Comment:
can you please give a short comment about these options? For example, can you specify multiple of these or only one can be specified per each run? Are they mutually exclusive or complimentary (e.g. if you give it for example 3utr does it mean it takes in cds+3utr or only exclusively 3utr)?

Reply:
Done! I added a more detailed description. If you specify ["cds","3utr"], it will return 2 sets of output files: one set containing genes in coding sequence regions, and the other set containing genes in 3' UTR region. I added the ability to provide a "custom" region file using the format specified here (https://oncodrivefml.readthedocs.io/en/latest/files.html#regions-file-format) in case the available regions files aren't sufficient

Reply:
>         # Available regions options: cds, lincrna, 3utr, 5utr, promoter_splice_sites, custom
        # You can specify more than one region at a time in list format, e.g ["cds","lincrna"]
        # This will result in 2 sets of output files, one set containing genes in coding sequence regions
        # And the other set containing long intergenic non-coding RNA results.
        # cds: Coding sequence regions
        # lincrna: Long intergenic non-coding RNA regions
        # 3utr: 3' UTR regions
        # 5utr: 5' UTR regions
        # promoter_splice_sites: Promoter regions and splice site regions
        # custom: Custom made regions file specified by user. More info for regions file format: 
        # https://oncodrivefml.readthedocs.io/en/latest/files.html#regions-file-format